src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunItems.java

```diff
@@ -350,6 +355,15 @@ private SlimefunItems() {}
         SLIME_CHESTPLATE_STEEL.addUnsafeEnchantments(slime);
         SLIME_LEGGINGS_STEEL.addUnsafeEnchantments(slime);
         SLIME_BOOTS_STEEL.addUnsafeEnchantments(slime);
+
+        Map<Enchantment, Integer> bee = new HashMap<>();
```

Comment:
I was gonna point out the irresponsibly chosen name, should have rather been "beeArmorEnchantments".
But then I realized the above armorsets use the exact same pattern ._.